src/server/modules/bulk/services/BulkService.ts

```diff
@@ -0,0 +1,96 @@
+import { injectable } from 'inversify'
+import paparse from 'papaparse'
```

Comment:
another small nit: usually it's imported as `Papa`, if not for consistency we could import it as `papaparse`? (instead of `paparse`)

Reply:
was grumpy about fixing this but here you go [6dfd46e](https://github.com/opengovsg/GoGovSG/pull/1993/commits/6dfd46e64663521905c4aa95625854512287dd5f) 🤣